### DISCUSSION TITLE
Labels automatically deleted when adding an issue to a project

### DISCUSSION THREAD

**Discussion by samjustus** [community/community#75437] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I am adding several issues to one of my github projects. Once I do so, all the labels are instantly completely cleared. Which makes them immediately disappear from my project view because they are based on labels for the most part. 

![image](https://github.com/community/community/assets/95646164/ad9b20b3-d535-4858-b1e7-febf2f28e804)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by KevinJoiner** [comment]
> testy poo

**Comment by samjustus** [comment]
i noticed that this is only when you add an issue to the project from within the project. If you add the issue to the project from within the issue, the labels seem to stay, at least after two tests
